repm/types.go

```diff
@@ -69,6 +69,20 @@ type delResponse struct {
 	Ok bool `json:"ok"`
 }
 
+
+type beginSyncRequest struct {
+	BatchPushURL  string `json:"batchPushURL"`
+	ClientViewURL string `json:"clientViewURL"`
```

Comment:
[fix?] This does not look right. The function just pass this into a function that "renames" it to `diffServerURL`. Should we just name the params: `batchUrl` & `diffUrl`?

Reply:
yes

Reply:
#76